maintainers/maintainer-list.nix

```diff
@@ -2928,6 +2928,12 @@
     githubId = 7047019;
     name = "Florent Becker";
   };
+  galagora = {
+    email = "lightningstrikeiv@gmail.com";
+    github = "galagora";
+    githubId = "45048741";
```

Comment:
This (`githubId`) should be an integer instead of a string.

Sorry, I missed that in my first review and CI doesn't seem to verify the types.

Reply:
Ok. Should I add @meutraa?

Reply:
Changed